pkg/services/ngalert/migration/alert_rule.go

```diff
@@ -6,66 +6,80 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/grafana/grafana/pkg/components/simplejson"
 	"github.com/grafana/grafana/pkg/infra/log"
 	legacymodels "github.com/grafana/grafana/pkg/services/alerting/models"
 	migmodels "github.com/grafana/grafana/pkg/services/ngalert/migration/models"
-	migrationStore "github.com/grafana/grafana/pkg/services/ngalert/migration/store"
 	ngmodels "github.com/grafana/grafana/pkg/services/ngalert/models"
 	"github.com/grafana/grafana/pkg/services/ngalert/store"
 	"github.com/grafana/grafana/pkg/tsdb/graphite"
 	"github.com/grafana/grafana/pkg/util"
 )
 
 const (
-	// ContactLabel is a private label created during migration and used in notification policies.
-	// It stores a string array of all contact point names an alert rule should send to.
-	// It was created as a means to simplify post-migration notification policies.
-	ContactLabel = "__contacts__"
+	// ContactLabelTemplate is a private label added to a rule's labels to route it to the correct migrated
+	// notification channel.
+	ContactLabelTemplate = "__contacts_%s__"
```

Comment:
I think this one also should be in `ngalert/models`

Reply:
I think we should add some hint to the UI page that says that such labels are not included into the notification and used only for routing purpose.